### PR TITLE
chore: 🤖 update lumberjack base config to angular v11

### PR DESCRIPTION
**Pull request by NachoVazquez**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

The example application is using angular v10 and the workspace is following the matching configuration.

Issue Number: N/A

## What is the new behavior?

The workspace and the application are now in the Angular v11 form.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: libs/ngworker/lumberjack/http-driver/src/lib/http-driver-root.module.ts [hunk 1]
```diff
@@ -37,7 +37,6 @@ export class HttpDriverRootModule {
     @SkipSelf()
     @Inject(HttpDriverRootModule)
     maybeNgModuleFromParentInjector: HttpDriverRootModule = null as any
-    // tslint:enable: no-any no-null-keyword
```
Comment (LayZeeDK) [inline libs/ngworker/lumberjack/http-driver/src/lib/http-driver-root.module.ts:40]: Why was this removed?
```suggestion
    // tslint:enable: no-any no-null-keyword
```

File: libs/ngworker/lumberjack/http-driver/src/lib/http-driver.module.spec.ts [hunk 2]
```diff
@@ -60,6 +60,7 @@ const createHttpDriver = (
 
   return httpDriver;
 };
+
```
Comment (LayZeeDK) [inline libs/ngworker/lumberjack/http-driver/src/lib/http-driver.module.spec.ts:63]: No need for this blank line between constant references even when they reference function expressions. Unless Prettier made you do it 😄
```suggestion
```

**Review by LayZeeDK — CHANGES_REQUESTED** [review]
No changes in TypeScript configurations for workspace, application, or libraries?

**Review by LayZeeDK — DISMISSED** [review]

**Review by LayZeeDK — COMMENTED** [review]
No changes to TypeScript configurations of library projects?

**Comment by NachoVazquez** [comment]
> No changes to TypeScript configurations of library projects?

I will double check.

File: libs/ngworker/lumberjack/tsconfig.lib.json [hunk 3]
```diff
@@ -5,6 +5,7 @@
     "outDir": "../../../out-tsc/lib",
     "target": "es2015",
```
Comment (LayZeeDK) [inline libs/ngworker/lumberjack/tsconfig.lib.json:6]: What's the `target` in a fresh Angular 11 workspace?

File: libs/ngworker/lumberjack/tsconfig.lib.json [hunk 4]
```diff
@@ -5,6 +5,7 @@
...
     "declaration": true,
+    "declarationMap": true,
     "inlineSources": true,
     "types": [],
     "lib": ["dom", "es2018"]
```
Comment (LayZeeDK) [inline libs/ngworker/lumberjack/tsconfig.lib.json:11]: What's the `lib` in a fresh Angular 11 workspace?

File: libs/ngworker/lumberjack/tsconfig.lib.json (hunk 3 above)
Comment (NachoVazquez) [reply libs/ngworker/lumberjack/tsconfig.lib.json:6]: The same.

File: libs/ngworker/lumberjack/tsconfig.lib.json (hunk 4 above)
Comment (NachoVazquez) [reply libs/ngworker/lumberjack/tsconfig.lib.json:11]: I would take it out then.

File: apps/lumberjack-app/src/test.ts [hunk 5]
```diff
@@ -1,8 +1,8 @@
 // This file is required by karma.conf.js and loads recursively all the .spec and framework files
-
+import 'zone.js/dist/zone-testing';
```
Comment (LayZeeDK) [inline apps/lumberjack-app/src/test.ts:2]: Where is import 'zone.js/dist/zone';

Comment (NachoVazquez) [reply apps/lumberjack-app/src/test.ts:2]: What do you mean?

Comment (LayZeeDK) [reply apps/lumberjack-app/src/test.ts:2]: The other test.ts files have this line before zone-testing
import 'zone.js/dist/zone';

Comment (NachoVazquez) [reply apps/lumberjack-app/src/test.ts:2]: I don't think I removed it, but it looks that it is not needed.

**Review by LayZeeDK — APPROVED** [review]
Great job!

**Pull request merged**